src/type/preload.ts

```diff
@@ -259,6 +259,16 @@ export type SavingSetting = {
   audioOutputDevice: string;
 };
 
+export type EngineSetting = Record<string, EngineSettingRecord>;
+
+export const engineSettingRecord = z.object({
+  useGpu: z.union([z.boolean(), z.literal("inherit")]).default("inherit"),
+  outputSamplingRate: z
+    .union([z.number(), z.literal("inherit"), z.literal("engineDefault")])
+    .default("inherit"),
+});
+export type EngineSettingRecord = z.infer<typeof engineSettingRecord>;
```

Comment:
デフォルト値設定する方法思いつきました！
```suggestion
export const defaultEngineSettingRecord = {
  useGpu: "inherit" as const,
  outputSamplingRate: "inherit" as const,
};
export const engineSettingRecord = z
  .object({
    useGpu: z.union([z.boolean(), z.literal("inherit")]),
    outputSamplingRate: z.union([
      z.number(),
      z.literal("inherit"),
      z.literal("engineDefault"),
    ]),
  })
  .default(defaultEngineSettingRecord);
export type EngineSettingRecord = z.infer<typeof engineSettingRecord>;
```

Reply:
調べてみました、空オブジェクトをパースするのが一番楽そうなのでFIXME消そうと思います（[colinhacks/zod#213](https://github.com/colinhacks/zod/issues/213)）